CRM/Upgrade/Incremental/sql/5.40.alpha1.mysql.tpl

```diff
@@ -13,3 +13,12 @@ UPDATE civicrm_option_value SET name = 'File'
 
 ALTER TABLE civicrm_mailing
 MODIFY COLUMN `created_date` timestamp NULL DEFAULT CURRENT_TIMESTAMP COMMENT 'Date and time this mailing was created.';
+
+-- https://lab.civicrm.org/dev/core/-/issues/2122
+
+ALTER TABLE `civicrm_event` ADD COLUMN `event_tz` text NULL DEFAULT NULL COMMENT 'Event\'s native time zone',
```

Comment:
Similarly do this in php instead using the convenience function that checks if it already exists, e.g. https://github.com/civicrm/civicrm-core/blob/4fbe460cd4a17d49bd76b82bde6335ecf861dce6/CRM/Upgrade/Incremental/php/FiveThirtySeven.php#L70